cyclops-ui/src/components/pages/Modules/Modules.tsx

```diff
@@ -227,14 +363,20 @@ const Modules = () => {
           </Button>
         </Col>
       </Row>
-      <Row gutter={[40, 0]}>
-        <Col span={18}>
+
+      <Row gutter={[10, 0]}>
+        <Col span={5}>
           <Input
             placeholder={"Search modules"}
-            style={{ width: "30%" }}
+            style={{ width: "100%" }}
             onChange={handleSearch}
           ></Input>
         </Col>
+        <Col span={2}>
+          <Popover content={resourceFilterPopover()} trigger={["click"]}>
+            <Button>{"Module Type "}</Button>
+          </Popover>
+        </Col>
```

Comment:
Can you make this button just an extension of the search bar like in this [example](https://codepen.io/pen?&editors=001)

Reply:
I think link is not working

Reply:
Oh yeah sorry, I can't share codepen links. You can check out [antd input](https://ant.design/components/input) and how to add the search icon to the right

<img width="483" alt="Screenshot 2024-07-22 at 22 26 22" src="https://github.com/user-attachments/assets/38c21be0-9c88-4f67-a0c7-79d8c08b5210">

Reply:
Changed it and let me know if CSS changes are required